databaker/loaders/xlsx.py

```diff
@@ -0,0 +1,165 @@
+                for name in self.workbook.sheetnames]
+
+
+class XLSXRowSet(RowSet):
+    """ Excel support for a single sheet in the excel workbook. Unlike
+    the CSV row set this is not a streaming operation. """
+
+    def __init__(self, name, sheet, window=None):
+        self.name = name
+        self.sheet = sheet
+        self.window = window or 1000
+        super(XLSXRowSet, self).__init__(typed=True)
+
+    def raw(self, sample=False):
+        for row_no, row in enumerate(self.sheet.iter_rows()):
+            row_of_cells = []
+            for col_no, openpyexcel_cell in enumerate(row):
+                row_of_cells.append(
+                    XLSXCell.from_openpyexcel(openpyexcel_cell,
+                            self.sheet, col_no, row_no))
+            yield row_of_cells
+
+
+class XLSXCell(Cell):
+
+    @staticmethod
+    def from_openpyexcel(openpyexcel_cell, sheet, col, row):
+        value = openpyexcel_cell.value
+
+        # Note: xls files only recognise type Float for numbers
+        # so cast integers to keep consistant typing across table loaders
+        if isinstance(value, int):
+            value = float(value)
+
+        # Note: Openpyxl registers whitespace as 'None' whereas xlrd uses 
+        # an empty string (''). Changing None values to empty strings here
+        # allows us to use xypaths .is/is_not_whitespace() functions.
+        if value == None:
+            value = ''
+
+        cell_type = get_data_type(openpyexcel_cell)
+
+        messy_cell = XLSXCell(value, type=cell_type)
+        messy_cell.sheet = sheet
+        messy_cell.openpyexcel_cell = openpyexcel_cell
+        messy_cell.xlrd_pos = (row, col) 
+
+        return messy_cell
+
+    @property
+    def topleft(self):
+        return self.properties.cell.topleft
+
+    @property
+    def properties(self):
+        return XLSXProperties(self)
+
+class XLSXProperties(CoreProperties):
+    KEYS = ['bold', 'italic', 'underline', 'blank', 'strikeout', 
+            'font_name', 'size', 'any_border', 'all_border', 
+            'richtext', 'a_date', 'formatting_string']
+
+    def __init__(self, cell):
+        self.cell = cell
+        self.merged = {}
+
+    def get_bold(self):
+        return self.cell.openpyexcel_cell.font.bold
+
+    def get_italic(self):
+        return self.cell.openpyexcel_cell.font.italic
+
+    def get_underline(self):
+        return self.cell.openpyexcel_cell.font.underline
+
+    def get_blank(self):
+        """Note that cells might not exist at all.
+           Behaviour for spanned cells might be complicated: hence this function"""
+        return self.cell.value == ''
+        
+    def get_strikeout(self):
+        return self.cell.openpyexcel_cell.font.strike
+
+    def get_font_name(self):
+        return self.cell.openpyexcel_cell.font.name
+
+    def get_size(self):
+        return self.cell.openpyexcel_cell.font.size
+
+    def get_any_border(self):
+        #b = self.cell.openpyexcel_cell.border
```

Comment:
we probably need to remove the commented out lines, unless we're keeping them for some reason